pkg/vrct/vrctFs/file_info.go

```diff
@@ -6,11 +6,12 @@ import (
 )
 
 type FileInfo struct {
-	name     string
-	size     int64
-	fileMode fs.FileMode
-	modTime  time.Time
-	isDir    bool
+	name    string
+	size    int64
+	mode    fs.FileMode
+	modTime time.Time
+	isDir   bool
+	sys     any
```

Comment:
What is this `sys` field and why does it have `any` type?

Reply:
[official os.Stat](https://pkg.go.dev/os#File.Stat)